src/type.ts

```diff
@@ -1,26 +1,25 @@
-import { Geometry } from 'geojson';
+import { Feature, FeatureCollection, Geometry } from 'geojson';
 
-export interface Feature {
-    type: string;
-    id: string;
-    geometry: Geometry;
-    properties : FeatureProperties;
-}
+export type OsmGoFeatureCollection = FeatureCollection<Geometry, FeatureProperties>;
+export type OsmGoFeature = Feature<Geometry, FeatureProperties>;
 
-
-interface FeatureProperties{
+export interface FeatureProperties{
     hexColor: string;
     icon: string;
-    id: string;
```

Comment:
@Binnette Please have a look at this. The existing code compars IDs in a way that is only allowed for numbers. So I'm pretty sure that only numbers are stored in the `id` property which is why I've updated this property to be of type `number`. Would be great if you can confirm this.

Reply:
hmm, yes they are integers but in text format ('1234') 
To really have integers in the data, you have to modify the "osmToOsmgo" converter, with a "ParseInt(xx.id)" line 632, 662, 715

Reply:
c40a7a300892dcf642752119e79da6ddd8ec0e2b d04c063de844123401b9f9f390fc706d922b97cc : propeties.id is now a true integer